FriendSchema.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+mongoose.connect("mongodb://localhost/friendsdb")
```

Comment:
We can use Promises here instead of Mongoose's `on` and `once` methods:

```
mongoose.connect("mongodb://localhost/friendsdb")
  .then(() => console.log('connected to friends database'))
  .catch(err => console.log(`error connecting to database: ${err.message}`))
```